odl/discr/grid.py

```diff
@@ -256,6 +256,16 @@ def max_pt(self):
         """
         return np.array([vec[-1] for vec in self.coord_vectors])
 
+    @property
+    def ideg(self):
+        """Indices of the degenerate axes of this grid."""
+        return self.__ideg
+
+    @property
+    def inondeg(self):
+        """Indices of the non-degenerate axes of this grid."""
+        return self.__inondeg
+
```

Comment:
In subclasses you would also write `self.__ideg`, no? As long as we don't use `other.__ideg`.